pkg/scheduler/scheduler.go

```diff
@@ -466,7 +466,9 @@ func (sched *Scheduler) scheduleOne() {
 					" No preemption is performed.")
 			} else {
 				preemptionStartTime := time.Now()
-				sched.preempt(pod, fitError)
+				if err := sched.preempt(pod, fitError); err != nil {
+					klog.Warningf("%v/%v preempting lower priority pods fail,%v", pod.Namespace, pod.Name, err)
```

Comment:
Shouldn't we increment scheduling failures here too? line https://github.com/kubernetes/kubernetes/pull/75369/files#diff-0260dd7e154d411be493ada0753dc547L479

Reply:
Yes, it will always run metrics.PodScheduleFailures.Inc().